data/mods/My_Sweet_Cataclysm/sweet_item_groups.json

```diff
@@ -118,5 +118,28 @@
       [ "smores", 2 ],
       [ "caff_gum", 2 ]
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "dessert",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "milkshake_choc", "prob": 15 },
+      { "item": "milkshake_deluxe_choc", "prob": 5 },
+      { "item": "icecream_choc", "prob": 10 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "fridgesnacks",
+    "subtype": "distribution",
+    "entries": [ { "item": "milk_choc", "prob": 25 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "fridge",
+    "//": "This is a terrible item group and should be phased out.  See SUS_fridge in SUS - > domestic.json for a more modern take on things.",
```

Comment:
The points of those item group is just to increase the quantity of sweets inside vanilla item groups. It's not really meant to contain sweet cataclysm exclusives item. You should add the chocolate milk variants to vanilla with a normal description and then use copy from to change the description for sweet cataclysm, and to maybe edit their frequency of spawn.